workshop/Lab1/README.md

```diff
@@ -0,0 +1,106 @@
+# Lab 1: Deploy an application with in-memory storage.
```

Comment:
this seems like a copy from kube101, I don't want to keep this in sync.

Reply:
+1 include a point to the next lab so they can just click-n-go and not have to think about it

Reply:
remove the "." at the end